src/PdfParser/Type/PdfIndirectObject.php

```diff
@@ -22,17 +22,17 @@ class PdfIndirectObject extends PdfType
     /**
      * Parses an indirect object from a tokenizer, parser and stream-reader.
      *
-     * @param int $objectNumberToken
-     * @param int $objectGenerationNumberToken
+     * @param int $objectNumber
+     * @param int $objectGenerationNumber
      * @param PdfParser $parser
      * @param Tokenizer $tokenizer
      * @param StreamReader $reader
-     * @return bool|self
+     * @return self|false
      * @throws PdfTypeException
      */
     public static function parse(
-        $objectNumberToken,
-        $objectGenerationNumberToken,
+        $objectNumber,
+        $objectGenerationNumber,
```

Comment:
Is the change here required? It may be a BC

Reply:
This is so low level that I cannot believe that anybody out there ever used this method with named arguments.